README.md

```diff
@@ -154,29 +154,34 @@ For multi-select inputs, when providing a custom `filterOptions` method, remembe
 	asyncOptions 		|	func		|	function to call to get options
 	autoload 			|	bool		|	whether to auto-load the default async options set
 	backspaceRemoves 	|	bool		|	whether pressing backspace removes the last item when there is no input value
+	cacheAsyncResults | bool    | enables the options cache for asyncOptions (default: `true`)
 	className 			|	string		|	className for the outer element
 	clearable 			|	bool		|	should it be possible to reset value
 	clearAllText 		|	string		|	title for the "clear" control when multi: true
 	clearValueText 		|	string		|	title for the "clear" control
 	delimiter 			|	string		|	delimiter to use to join multiple values
-	disableCache 		|	bool		|	disables the options cache for asyncOptions
```

Comment:
`disableCache` seems to have been replaced by `cacheAsyncResults`, see discussion on: #355
> I thought of cacheAsyncResults about a minute after I committed the change to disableCache... :smile: